pkg/roll/run_update.go

```diff
@@ -480,7 +484,11 @@ func (u *update) cleanupOldRC(ctx context.Context) {
 			"rc-deletion-txn"+u.ID().String(),
```

Comment:
we could probably make the transaction stuff return a special error type and only send this alert if the error is because the transaction got beyond 64 operations (consul's max). That was the intent here, all other errors don't make sense to alert for now that we retry them